index.Rmd

```diff
@@ -1,32 +1,41 @@
 --- 
 title: "Azure DevOps for Analysis"
+author: "Department for Education guidance"
+date: "`r Sys.Date()`"
 site: bookdown::bookdown_site
+output: bookdown::gitbook
 documentclass: book
 bibliography: [book.bib, packages.bib]
 biblio-style: apalike
 link-citations: yes
 description: "This is a learning resource for using Git and Azure DevOps (DevOps) for managing Analytical Projects."
+favicon: www/images/qa_logo.png
 ---
 
-# Preface {-}
+# Preface
 
 ## Overview
 
-This book aims act as a resource for analysts on how and why they should use version control.
+This book acts as a resource for analysts on how Git works and why it is useful for collaborative working and version control.
```

Comment:
I think the double "and" in this sentence makes it a bit cumbersome. For me a comma before the first "and" would make it clearer.